src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,114 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import './FindMovie.scss';
 
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+import { getMovie } from '../../api/api';
+
+export const FindMovie = ({ addMovie }) => {
+  const [movie, setMovie] = useState([]);
+  const [title, setTitle] = useState('');
+  const [error, setError] = useState(false);
+
+  const movieSearch = (movieName) => {
+    getMovie(movieName)
+      .then((data) => {
+        setMovie({
+          imdbId: data.imdbID,
+          title: data.Title,
+          description: data.Plot,
+          imgUrl: data.Poster,
+          imdbUrl: `https://www.imdb.com/title/${data.imdbID}/`,
+        });
+        setError(data.Error);
```

Comment:
As a variant you could check if `data.Response === 'False'` and then `setError(true)` if you are using boolean values for your `error`